source/nodejs/adaptivecards-designer-app/samples/Restaurant.data.json

```diff
@@ -0,0 +1,29 @@
+{
+	"@context": "http://schema.org/",
+	"@type": "LocalBusiness",
+	"name": "Malt \u0026 Vine",
+	"url": "https://www.yelp.com/biz/malt-and-vine-redmond",
+	"address": {
+		"addressLocality": "Redmond",
+		"addressRegion": "WA",
+		"streetAddress": "16851 Redmond Way",
+		"postalCode": "98052",
+		"addressCountry": "US"
+	},
+	"image": "https://s3-media1.fl.yelpcdn.com/bphoto/HD_NsxwaCTwKRxvOZs2Shw/ls.jpg",
+	"telephone": "+14258816461",
+	"aggregateRating": {
+		"reviewCount": 176,
+		"@type": "AggregateRating",
```

Comment:
INteresting -- are we saying we may have an alternate way with databinding to implement rating controls?

Reply:
No, this is just data. To display that data as a "rating" you need to have some sort of a rating control whether it's ChoiceSet or an explicit Rating control doesn't matter)

Reply:
got it! tx

---
In reply to: [297899768](https://github.com/microsoft/AdaptiveCards/pull/2995#discussion_r297899768) [](ancestors = 297899768)